topics/fair/tutorials/bioimage-REMBI/tutorial.md

```diff
@@ -0,0 +1,409 @@
+  
+time_estimation: "15m"
+
+key_points:
+- REMBI describes useful guidelines for bioimaging that can help unification and FARIfication of the data.
+
+tags:
+- fair
+- data management
+- bioimaging
+  
+priority: 5
+
+contributions:
+  authorship:
+    - wee-snufkin
+    - Laura190
+    - kkamieniecka
+    - poterlowicz-lab
+
+subtopic: fair-data
+
+requirements:
+  - type: "internal"
+    topic_name: fair
+    tutorials:
+      - fair-intro
+      - data-management
+      - bioimage-metadata
+
+follow_up_training:
+  - type: "internal"
+    topic_name: imaging
+---
+
+# Metadata guidelines for bioimaging data
+
+REMBI (Recommended Metadata for Biological Images) was proposed as a draft metadata guidelines to begin addressing the needs of diverse communities within light and electron microscopy. Currently, these guidelines are in draft form to encourage discussion within the community, but they provide a useful guide as to what metadata should be gathered to make your image data FAIR. They divide the metadata requirements into eight modules which further split into attributes - that seems to be a daunting task, doesn't it? But at the same time it's exciting news for the community! To find out more, have a look at the [REMBI article](https://www.nature.com/articles/s41592-021-01166-8).
+
+
+> <question-title></question-title>
+>
+> In the [REMBI paper](https://www.nature.com/articles/s41592-021-01166-8), the authors consider three potential user groups who require different metadata. Find out what are these three groups and their metadata requirements.
+>
+> > <solution-title></solution-title>
+> > The identified three user groups are: Biologists, Imaging scientists, Computer-vision researchers. 
+> > - A research biologist may be interested in the biological sample that has been imaged to compare it to similar samples that they are working with.
+> > - An imaging scientist may be interested in how the image was acquired so they can improve upon current image acquisition techniques.
+> > - A computer vision researcher may be interested in annotated ground-truth segmentations, that can be obtained from the image, so they can develop faster and more accurate  algorithms.
+> {: .solution}
+>
+{: .question}
+
+> <tip-title>Instructor Note</tip-title>
+>
+> If you're an instructor leading this training, you might ask people to work in small groups for this exercise and encourage the discussion. Ask group members to share which of the user groups they identify as and what metadata they would want.
+> 
+{: .tip}
+
+# Categories of metadata 
+REMBI covers different categories of metadata, such as: 
+- study
+- study component
+- biosample
+- specimen
+- image acquisition
+- image data
+- image correlation
+- analyzed data
+
+> Within each module, there are attributes that should be included to make the published data FAIR. We will explore all the modules and attributes suggested by REMBI and we'll show some examples as well. 
```

Comment:
(getting rid of the `>` at the start of the line should fix the errors, same for most of the other errors below)